lib/rules/no-fallthrough.js

```diff
@@ -39,6 +39,19 @@ function isReachable(segment) {
     return segment.reachable;
 }
 
+/**
+ * Checks whether a SwitchCase node is adjacent to the token after it
+ * @param {ASTNode} node - The case node to check
+ * @param {RuleContext} context - Rule context
+ * @returns {boolean} `true` if the case is adjacent to the next token
+ */
+function isAdjacentToNextToken(node, context) {
+    var sourceCode = context.getSourceCode();
+    var nextToken = sourceCode.getTokenAfter(node);
+
+    return nextToken && nextToken.loc.start.line <= node.loc.end.line + 1;
```

Comment:
Sorry, I'm not following. Can you explain why that would trigger `<` rather than `===`?